### ISSUE TITLE
Wear OS app doesn't sync with the phone app

### ISSUE THREAD

**Issue by AmaryllisVFX** [rozPierog/Cofi#141]
**Describe the bug**
The Wear OS app doesn't sync my recipes from the phone app and only displays the default recipes. The watch app also prompts me to install the phone app even though I have it installed already. If I open any of the default recipes the watch app crashes. I've tried reinstalling both the phone and watch apps multiple times along with trying the play store version and sideloading the app manually.

**To Reproduce**
1. Open the watch app
2. Open a recipe
3. Watch app crashes

**Expected behavior**
The watch app correctly syncs all the recipes from the phone app and is able to run the recipes without crashing.

**Smartphone:**
 - Device: POCO F1
 - Android: 13
 - App Version: 1.13.1

**Watch:**
- Device: Fossil Gen 5
- Wear OS: 2.38 H MR2
- App Version: 1.13.1

**Additional context**
The watch app used to work correctly with the initial release and stopped working with subsequent versions.

**Comment by rozPierog** [comment]
I'm sorry that has happened to you @AmaryllisVFX 
First of all I need to know if you have app from the play store installed on both your Poco and your Fossil. For sync to work the app needs to be from the same "source" so you either need to have it both from the play store or both from GitHub (and phone app should be a "play store" variant)
I've tested it on emulator (but only WearOS 3, because there are no ARM system images of wearOS 2) and it worked fine. I'll try to investigate it further.

Do you remember last working version? Or approximately when was it working?

**Comment by AmaryllisVFX** [comment]
I have the play store versions on both the phone and the watch. The watch app worked fine on 1.11.0

**Comment by rozPierog** [comment]
Okay, thanks. I'll investigate it, sorry for the inconveniences

**Comment by AmaryllisVFX** [comment]
No worries! Thank you for responding so quickly

**Comment by rozPierog** [comment]
@AmaryllisVFX I've managed to isolate and fix the issue with app crashing on WearOS 2, thanks for reporting it!  I've released new version and I hope that it will also fix your sync problems. If you want to install new version as apk
- go to https://github.com/rozPierog/Cofi/releases/tag/v1.13.2 and download ` app-playStore-release.apk ` and ` wearOS-release.apk `
- sideload ` wearOS-release.apk ` to your watch
- install `app-playStore-release.apk` on your phone

It's `in review` on the Play Store so it will take some time for it to be visible there.
